performance_tests/10zone1vav.idf

```diff
@@ -35,10 +35,12 @@
   Output:DebuggingData,0,0;
 
   ZoneCapacitanceMultiplier:ResearchSpecial,
+    1.,                      !- Name
```

Comment:
Well, the error does show so not sure what happened. There are also many warnings and severes in this file, maybe the transition will eliminate some or all of them. I still think if the zone were in a zone list and not found there would be no warning.

    ** Severe  ** ZoneCapacitanceMultiplier:ResearchSpecial="1." invalid Zone or ZoneList Name="1." not found.
    ** Severe  ** GetStagedDualSetpoint: Errors with invalid names in ZoneControl:Thermostat:StagedDualSetpoint objects.
    **   ~~~   ** ...These will not be read in.  Other errors may occur.
    **  Fatal  ** Errors getting Zone Control input data.  Preceding condition(s) cause termination.

The node list should be:

    NodeList,
    PVAV_Sys Supply Fan Upstream Nodes,  !- Name
    PVAV_Sys Mixed Air Outlet,  !- Node 1 Name
    PVAV_Sys Cooling Coil Outlet;  !- Node 2 Name